go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/jfrog/jfrog-cli-security
 
-go 1.22.3
+go 1.22.7
```

Comment:
sure we can we work with 22.7 when CLI is 23 right?
No need to upgrade further here?

Reply:
yeah, CLI is the last component.
We did not upgraded it. it came from `cli-core`